quupe-RN/js/navigation/NavigationLayout.js

```diff
@@ -76,8 +78,11 @@ const authSwitch = createSwitchNavigator(
   }
 )
 
+const signUpStack = createStackNavigator({ SignUp: SignUpContainer });
+
```

Comment:
I think, sign up and sign in can be part of the same stack, that way it will be easy to navigate from the sign in to sign up. Since sign up and sign in can be part of the same stack, we can name that stack authStack. 

Just for your reference: https://reactnavigation.org/docs/en/auth-flow.html